tools/ExtensionsMetadataGenerator/test/ExtensionsMetadataGeneratorTests/ExtensionsMetadataGeneratorTests.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.35" />
```

Comment:
Is there a reason for this referenced version downgrade?

Reply:
I ready that as 3.0.05, so I thought this would be an upgrade. I don't think WebJobs 3.0.5  as in (50) exists?